SingularityService/src/main/java/com/hubspot/singularity/resources/S3LogResource.java

```diff
@@ -235,7 +257,10 @@ private void checkS3() {
   @GET
   @Path("/task/{taskId}")
   @ApiOperation("Retrieve the list of logs stored in S3 for a specific task.")
-  public List<SingularityS3Log> getS3LogsForTask(@ApiParam("The task ID to search for") @PathParam("taskId") String taskId) throws Exception {
+  public List<SingularityS3Log> getS3LogsForTask(
+      @ApiParam("The task ID to search for") @PathParam("taskId") String taskId,
+      @ApiParam("Start timestamp (millis, 13 digit)") @QueryParam("start") long start,
```

Comment:
Not sure you can use `Optional` in `@QueryParam`, I think if the query param is not specified it will always come back as a null no matter what the class. I'll double check though